impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -369,13 +369,23 @@ define(function() {
           extension: {
             // Plot
             sizeAxisRatio: 1 / 5,
-            sizeAxisRatioTo: "height", // plot area client height
+            sizeAxisRatioTo: "minwidthheight",
```

Comment:
The null crosses at the top were being cut. That's the way I found to avoid https://github.com/webdetails/ccc/blob/master/package-res/ccc/plugin/scatter/scatter-plot-panel.js#L401-L407 and saw no side effects, but I was expecting your input on that.